apm-server/README.md

```diff
@@ -8,8 +8,8 @@ The design and code is less mature than official GA features and is being
 provided as-is with no warranties. Alpha features are not subject to the support
 SLA of official GA features (see [supported configurations][] for more details).
 
-**Warning**: This branch is used for development, please use [7.9.2][https://github.com/elastic/helm-charts/releases/tag/7.9.2] release for released version.
-
+**Warning**: This branch is used for development, please use the latest [7.9](https://github.com/elastic/helm-charts/releases)
+release for released version.
```

Comment:
Maybe keeping this statement in a single line would make easier to automate removing it during release and adding it back after releases

Reply:
The intent with that PR is that we will **not** add/remove that warning statement for / after the release. Do you foresee this being an issue?

Reply:
Yes this is an issue because this statement will be displayed in [Artifact Hub](https://artifacthub.io/packages/helm/elastic/elasticsearch) and will make user think that released charts are for development.